src/Modules/UngradedModules.jl

```diff
@@ -437,9 +438,28 @@ Construct `ModuleGens` from an array of Oscar free module elements, specifying t
 """
 function ModuleGens(O::Vector{<:FreeModElem}, F::FreeMod{T}) where {T}
   #SF = singular_module(F)
+  #return ModuleGens{T}(O, F, SF)
   return ModuleGens{T}(O, F)
 end
 
+@doc Markdown.doc"""
+    ModuleGens(O::Vector{<:FreeModElem}, F::FreeMod{T}, ordering::ModuleOrdering) where {T}
+
+Construct `ModuleGens` from an array of Oscar free module elements, specifying the Oscar free module.
+Moreover, the ordering is defined by `ordering1` and `ordering2`. Allowed symbols are those that are 
```

Comment:
There are no such arguments. Maybe you meant this?
```suggestion
Moreover, the ordering is defined by `ordering`. Allowed symbols are those that are 
```